stockfish/models.py

```diff
@@ -550,6 +550,60 @@ def set_depth(self, depth_value: int = 2) -> None:
         """
         self.depth = str(depth_value)
 
+    def get_what_is_on_square(self, square: str) -> str:
```

Comment:
Maybe it would be better to return enum?

```python
from enum import Enum

class Piece(Enum):
    BLACK_KING = "black king"
    WHITE_ROCK = "white rock"
    ...
```

Also I think it would be better to return None, when the square is empty.